server/segment_rpc.go

```diff
@@ -348,7 +348,7 @@ func SubmitSegment(sess *BroadcastSession, seg *stream.HLSSegment, nonce uint64)
 		monitor.LogSegmentTranscoded(nonce, seg.SeqNo, transcodeDur, tookAllDur, common.ProfilesNames(sess.Profiles))
 	}
 
-	glog.Info("Successfully transcoded segment ", seg.SeqNo)
+	glog.Infof("Successfully transcoded segment segName=%s seqNo=%d", seg.Name, seg.SeqNo)
```

Comment:
Should we use `mid=%s seq=%s", sess.ManifestID, sess.SeqNo` here instead? (Or the mid in addition to the segName which is basically a URL if I recall.)

As an aside, it'd probably be good to settle on conventions for what and how to tag fields in logs.

Reply:
Changed.
Yes, it would be good.